Automattic/pocket-casts-ios
# [Enhancement] Replace Subscribe with Follow

As discussed in our P2 we want to update some texts in few screens, to replace the word `Subscribe` with `Follow`

| Podcast Page | Podcast Page Unsubscribe | Podcast Settings | Podcast Page Unsubscribe Download |
| :------: | :------: | :------: | :------: |
| ![RocketSim_Screenshot_iPhone_16_6 1_2024-10-29_16 31 23](https://github.com/user-attachments/assets/163821db-571b-4477-8edd-56c8d5c4bb54) | ![RocketSim_Screenshot_iPhone_16_6 1_2024-10-29_16 31 13](https://github.com/user-attachments/assets/35836865-3d87-47f8-815d-521be7459ad1) | ![RocketSim_Screenshot_iPhone_16_6 1_2024-10-29_16 31 59](https://github.com/user-attachments/assets/87face04-feaa-4b29-891d-d7378ad4ab54) | ![RocketSim_Screenshot_iPhone_16_6 1_2024-10-30_09 37 02](https://github.com/user-attachments/assets/6d5fcac6-6e99-4f27-ba3a-24c5be35244f)
| Filters All Podcast | Apply Criteria | Export Settings |
| ![RocketSim_Screenshot_iPhone_16_6 1_2024-10-29_16 32 27](https://github.com/user-attachments/assets/ede6fae3-08d2-49d8-8e28-99d84d46ac9b) | ![RocketSim_Screenshot_iPhone_16_6 1_2024-10-29_16 30 55](https://github.com/user-attachments/assets/c840c519-6b1a-4e4b-a76a-52ca567eff22) | ![RocketSim_Screenshot_iPhone_16_6 1_2024-10-29_16 32 44](https://github.com/user-attachments/assets/3bf9da1e-8717-4fb2-9f5e-99c06d0a748a) |


## To test

- CI musty be 🟢 
- Run the app
- Check the above screens
- Switch off the FF `useFollowNaming`
- Run the app to see if the changes are reverted

## Checklist
